website/description_issues.qmd

```diff
@@ -25,3 +25,19 @@ Description: The goal of 'readr' is to provide a fast and friendly way to
     to flexibly parse many types of data found in the wild, while still
     cleanly failing when data unexpectedly changes.
 ```
+
+------------------------------------------------------------------------
+
+# Explaining Acronyms
+
+## Problem
+
+The automatic spell check has flagged an acronym as incorrect and results in a NOTE.
+
+## Solution
+
+Document all non-obvious acronyms in the [cran-comments.md](https://usethis.r-lib.org/reference/use_cran_comments.html) file to facilitate the CRAN team review.
+
+### Details
+
+Most acronyms that are not widely known should be explained. For example, you don’t have to explain *OLS*, *SEO*, or *DNA* but explanations should be provided for: *MEFM* or *OCCDS*
```

Comment:
Just a small addition: (maybe as a "tip") If unsure, explain the acronym (a package which explains "DNA" won't be archived, but one which does not explain "MEFM" will.

Reply:
I think that is good to articulate so folks are not in limbo of determining if their acronym is widely known.